teacher.rb

```diff
@@ -0,0 +1,13 @@
+require './person'
+
+class Teacher < Person
+  def initialize(age, specialization, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
+    @specialization = specialization
+  end
+
+  def can_use_services?
+    super()
+    true
+  end
```

Comment:
- [ ] Here you don't need to invoke the **super** function because this method overrides `can_use_services?` so it **always** returns `true`.`